Python/pywarpx/picmi.py

```diff
@@ -1218,6 +1221,7 @@ def initialize_inputs(self):
         self.diagnostic.intervals = self.period
         self.diagnostic.diag_type = 'Full'
         self.diagnostic.format = 'checkpoint'
+        self.diagnostic.file_min_digits = file_min_digits
```

Comment:
Fails with:
```
NameError: name 'file_min_digits' is not defined
```

I don't understand why, help wanted @dpgrote :)

Reply:
It needs to be `self.file_min_digits`.

Reply:
dang, of course. thanks :)